tools/ashlar/macros.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<macros>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="@TOOL_VERSION@">ashlar</requirement>
```

Comment:
if we go docker only we can remove the package line

Reply:
Keeping package will make it compatible for users that want to stick with conda.